Packs/CommonScripts/Scripts/ZipFile/ZipFile_test.py

```diff
@@ -12,3 +12,11 @@
 @pytest.mark.parametrize(('input_name', 'output_name'), ESCAPE_CHARACTERS_PACK)
 def test_escape_characters_in_file_name(input_name, output_name):
     assert escape_illegal_characters_in_file_name(input_name) == output_name
+
+
+def test_compress_multiple_with_password():
+    compress_multiple_with_password(
```

Comment:
lets check that we cant open this file without the password, and that we can with the password

Reply:
done